requirements-tests39.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
The proper way to update requirements is `tox -e update_requirements39`.

``` diff
diff --git a/requirements-tests39.txt b/requirements-tests39.txt
index 8bd2b1d..fbc3f3c 100644
--- a/requirements-tests39.txt
+++ b/requirements-tests39.txt
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.11
+# This file is autogenerated by pip-compile with Python 3.9
 # by the following command:
 #
 #    pip-compile --no-emit-index-url --output-file=requirements-tests39.txt requirements-tests39.in
@@ -22,6 +22,8 @@ docutils==0.18.1
     # via
     #   doc8
     #   restructuredtext-lint
+exceptiongroup==1.1.3
+    # via pytest
 flake8==4.0.1
     # via -r requirements-tests39.in
 iniconfig==1.1.1
@@ -77,11 +79,16 @@ stevedore==3.5.0
 toml==0.10.2
     # via autopep8
 tomli==2.0.1
-    # via coverage
+    # via
+    #   coverage
+    #   pylint
+    #   pytest
 tomlkit==0.11.6
     # via pylint
 typing-extensions==4.4.0
-    # via astroid
+    # via
+    #   astroid
+    #   pylint
 untokenize==0.1.1
     # via docformatter
 wrapt==1.14.1
```

Reply:
Thanks, updated.